hummingbot/client/ui/parser.py

```diff
@@ -89,12 +89,32 @@ def load_parser(hummingbot, command_tabs) -> [ThrowingArgumentParser, Any]:
                                 dest="precision", help="Level of precions for values displayed")
     history_parser.set_defaults(func=hummingbot.history)
 
-    gateway_parser = subparsers.add_parser("gateway", help="Gateway API configurations")
-    gateway_parser.add_argument("option", nargs="?", choices=("create", "config", "connect", "generate-certs",
-                                                              "status", "test-connection", "start", "stop"), help="Gateway configuration choices")
-    gateway_parser.add_argument("key", nargs="?", default=None, help="Name of the parameter you want to change")
-    gateway_parser.add_argument("value", nargs="?", default=None, help="New value for the parameter")
-    gateway_parser.set_defaults(func=hummingbot.gateway)
+    gateway_create_parser = subparsers.add_parser("gateway create", help="Create gateway docker container instance")
```

Comment:
The gateway commands shouldn't be at the same level as the `history`, `start`, etc. commands. You should add a gateway parser and then add subparsers under the gateway parser.

Reply:
Do you mean we should have a `gateway_parser` function which is similar to that `load_parser` function?

Reply:
No, I'm saying you do something like this.

```
gateway_parser = subparsers.add_parser("gateway", ...)
gateway_subparsers = gateway_parser.add_subparsers()
gateway_create_parser = gateway_subparsers.add_parser("create", ...)
...
```

Reply:
You can factor that out to another `load_gateway_parser()` function if you like. But the important thing is to create the parsers in a tree structure and allow the gateway commands to reside in their appropriate level in the tree, all with the appropriate help messages.